services/blockchain-indexer/config.js

```diff
@@ -79,24 +79,19 @@ config.operations = {
 config.networks = [
 	{
 		name: 'mainnet',
-		address: '',
+		chainID: '00000000',
 		serviceURL: 'https://service.lisk.com',
-		identifier: '4c09e6a781fc4c7bdb936ee815de8f94190f8a7519becd9de2081832be309a99',
-		genesisBlockUrl: 'https://downloads.lisk.com/lisk/mainnet/genesis_block.json.tar.gz',
```

Comment:
Do we not use the `genesisBlockUrl` anymore within the indexer?

Reply:
Yes, we are not using `genesisBlockUrl `  anywhere in the `indexer/dataservice` service